cocos/gfx/webgl/webgl-texture.ts

```diff
@@ -94,30 +98,32 @@ export class WebGLTexture extends Texture {
                 isSwapchainTexture: isSwapchainTexture || false,
             };
 
-            if (!this._gpuTexture.isSwapchainTexture) {
-                WebGLCmdFuncCreateTexture(WebGLDeviceManager.instance, this._gpuTexture);
-                WebGLDeviceManager.instance.memoryStatus.textureSize += this._size;
+            if (!self._gpuTexture.isSwapchainTexture) {
+                WebGLCmdFuncCreateTexture(instance, self._gpuTexture);
+                instance.memoryStatus.textureSize += self._size;
             }
 
-            this._viewInfo.texture = this;
-            this._viewInfo.type = info.type;
-            this._viewInfo.format = info.format;
-            this._viewInfo.baseLevel = 0;
-            this._viewInfo.levelCount = info.levelCount;
-            this._viewInfo.baseLayer = 0;
-            this._viewInfo.layerCount = info.layerCount;
+            thisViewInfo.texture = self;
```

Comment:
Adding varible `thisViewInfo` to avoid `this._viewInfo` being inovked many times.